resource_booking_combination_auto_create/wizards/resource_booking_combination_wizard.py

```diff
@@ -182,9 +182,10 @@ def create_combination(self):
         resource_combination_model = self.env["resource.booking.combination"]
         existing_combinations = resource_combination_model.search(domain)
         resource_combination = resource_combination_model.browse()
-        for resource_combination in existing_combinations:
+        for combination in existing_combinations:
             # use first matching combination
-            if len(resource_combination.resource_ids) == len(resources):
+            if len(combination.resource_ids) == len(resources):
+                resource_combination = combination
                 break
```

Comment:
Can't you use `(combination.resource_ids & resources) == resources`?

Reply:
the domain already ensures that the combination contains all the needed resources, but it can contain more. this is why comparing the length is enough and slightly more efficient.

Reply:
ha, that's really cool